core/src/main/scala/akka/persistence/cassandra/query/EventsByTagStage.scala

```diff
@@ -743,14 +752,23 @@ import scala.compat.java8.FutureConverters._
                 explicitGapDetected))))
       }
 
+      private def totalMissing(): Long = {
+        val missing = getMissingLookup()
+        missing.remainingMissing.values.foldLeft(0L)(_ + _.size)
+      }
+      private def failTooManyMissing(total: Long): Unit = {
+        failStage(
+          new RuntimeException(s"$total missing tagged events for tag [${session.tag}]. Failing without search"))
+      }
+
       @tailrec def tryPushOne(): Unit =
         stageState.state match {
           case QueryResult(rs) if isAvailable(out) =>
             if (isExhausted(rs)) {
               queryExhausted()
             } else if (rs.remaining() == 0) {
               log.debug("[{}] Fetching more", stageUuid)
-              fetchMore(rs)
+
```

Comment:
This `if` branch just writes to log and nothing to do now? 🤔

Reply:
That was was a mistake (hence all the test failures). Thanks!